### PR TITLE
[OKD-Core] Add VMs to Project Overview

### PR DESCRIPTION
**Pull request by rawagner**
web-ui-components PR https://github.com/kubevirt/web-ui-components/pull/245

### CODE REVIEW

**Comment by rawagner** [comment]
vm_overview

**Comment by rawagner** [comment]
Added status to VM overview item
vm_overview_status

**Comment by suomiy** [comment]
ci failed

File: frontend/public/kubevirt/components/overview/vm-overview.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
```
Comment (mareklibra) [inline frontend/public/kubevirt/components/overview/vm-overview.jsx]: What about:
import React from 'react';

**Comment by mareklibra** [comment]
Is behaviour as expected?
I can newly see list of groups, one per a VM. Each group contains single item - the VM.

01_status

**Comment by rawagner** [comment]
> Is the behaviour as expected?
> I can newly see list of groups, one per a VM. Each group contains single item - the VM.

yes, it is expected - you have Group By - Application which groups resources by `app=<something>` label. If you change it to Group By Resource you will groups of VMs/Pods etc.

**Pull request merged**